spec/features/visitors/home_page_spec.rb

```diff
@@ -6,4 +6,10 @@
     expect(page).to have_content I18n.t 'main.title'
   end
 
+  scenario 'Should redirect to Website map', :search do
+    visit root_path
+    find(:xpath, "/html/body/footer/div[2]/div/div[2]/ul/li[3]/a").click
```

Comment:
Why are you using ```find(:xpath, "/html/body/footer/div[2]/div/div[2]/ul/li[3]/a").click```, instead of ```click_link "Mapa web" ```.... Are you having trouble with capybara?

Also we can deactivate Sunspot on this test removing **:search** metadata from scenario.

Reply:
Is fixed now, It was an error by blanks with "click_link" .